middleware.md

```diff
@@ -61,7 +61,6 @@ It's best to envision middleware as a series of "layers" HTTP requests must pass
 > [!NOTE]  
 > All middleware are resolved via the [service container](/docs/{{version}}/container), so you may type-hint any dependencies you need within a middleware's constructor.
 
-<a name="before-after-middleware"></a>
 <a name="middleware-and-responses"></a>
 #### Middleware and Responses
```

Comment:
The two links results bad HTML (nested links) and the first link does not work in Chrome or Safari. It works in Firefox, but only if it is opened to directly and not navigated to after page load.